public/js/commentForm.js

```diff
@@ -0,0 +1,44 @@
+var commentForm = (function(){
+
+    var el = {
+        $comment_input: $('#comment_input'),
+        $comment_buttons_container: $('.comment-form__button-container'),
+        $github_signin: $('#github_signin')
+    };
+
+    var bindUIElements = function() {
+        el.$comment_input.on('focus', showCommentButton);
+        el.$comment_input.on('blur', hideCommentButton);
+        el.$github_signin.on('click', handleSignin);
+    };
+
+    var init = function() {
+        bindUIElements();
+        refillPreviousComment();
+    };
+
+    var handleSignin = function (e) {
+        var comment = el.$comment_input.val();
+        localStorage.setItem('gist_comment', comment);
+    };
+
+    var showCommentButton = function () {
+        el.$comment_buttons_container.addClass('active');
+    };
+
+    var hideCommentButton = function () {
+        if(el.$comment_input.val() === "") el.$comment_buttons_container.removeClass('active');
```

Comment:
Just because I'm picky, can you add a space after the `if` and add braces and split this out to multiple lines? :D